### ISSUE TITLE
Runtime error on start

### ISSUE THREAD

**Issue by tohvanah** [LoopKit/CGMBLEKit#136]
I'm not a Swift developer, and rarely work in Xcode; I'm just trying to build the ResetTrasmitter app. Any help is appreciated.

macOS 10.14.6
Xcode 11.2
Tried on iOS 11.4 and iOS 12.3.1

The following exception occurs on the first run of the app every time. I have even tried old releases (2.1.1), and other branches (xcode11, dev).

`2019-10-31 23:18:28.525462-0400 ResetTransmitter[299:19291] *** Terminating app due to uncaught exception 'NSInvalidUnarchiveOperationException', reason: 'Could not instantiate class named _UITextLayoutView'
*** First throw call stack:
(0x184d8ed8c 0x183f485ec 0x184d8ec6c 0x18f16f5fc 0x18f16f8b0 0x18f16f470 0x18edc9be4 0x18ede2860 0x18f5bbb18 0x10493b148 0x10493b208 0x18f00dfec 0x18f16f738 0x18f16f8b0 0x18f16f470 0x18edc9be4 0x18f1c8820 0x18f16f738 0x18f16f8b0 0x18f16f470 0x18edc9be4 0x18efdbd5c 0x18f16f738 0x18f16f470 0x18f305080 0x18f16f738 0x18f16f8b0 0x18f16f470 0x18f3055d8 0x18f16f738 0x18f16f8b0 0x18f16f470 0x18f3058f4 0x18f16f738 0x18f16f470 0x18f00dc94 0x18f16f738 0x18f16f8b0 0x18f16f470 0x18f00cfe4 0x18ee3eaa8 0x18ea82074 0x18e9f2850 0x18e9a6b14 0x18ea54e1c 0x18ea542d8 0x18ea53c14 0x18ea536a4 0x18ea473a8 0x18e99f6f4 0x188f15fec 0x188f1a17c 0x188e86830 0x188eae364 0x18ed89680 0x184d372bc 0x184d36a7c 0x184d347b0 0x184c54da8 0x186c39020 0x18ec71758 0x10492c2d8 0x1846e5fc0)
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb) `

**Comment by tohvanah** [comment]
I was able to successfully build and run the master branch after updating the iOS 11.4 device (iPhone SE) to 13.2. (I also had to edit the info.plist to stop the immediate crashing re #132 ) It's odd that it crashed for iOS 11 & 12, and it hasn't been updated to run on iOS 13... Is it an Xcode thing?

**Comment by MaximSkorynin** [comment]
Yes, the error is related to the new version of the Xcode 11.2

**Comment by tohvanah** [comment]
Ok thanks. I see you can [download older versions of Xcode](https://developer.apple.com/download/more/?name=Xcode), what is recommended/stable at this point?

**Comment by MaximSkorynin** [comment]
Yes, but then you will not be able to test the application on devices where iOS 13 is installed (

**Comment by marionbarker** [comment]
Please close this Issue by scrolling to the bottom of the screen at GitHub and tapping on "Close issue"

BR/Marion
